tests/motion_setpoints.py

```diff
@@ -37,82 +44,86 @@
 
 POSITION_SAMPLE1 = "Sample1"
 POSITION_SAMPLE2 = "Sample2"
-MOTOR_POSITION_SAMPLE1_COORD1 = 3
-MOTOR_POSITION_SAMPLE1_COORD2 = -2
-MOTOR_POSITION_SAMPLE2_COORD1 = 1
-MOTOR_POSITION_SAMPLE2_COORD2 = -3
+MOTOR_POSITION_SAMPLE1_COORD0 = 3
+MOTOR_POSITION_SAMPLE1_COORD1 = -2
+MOTOR_POSITION_SAMPLE2_COORD0 = 1
+MOTOR_POSITION_SAMPLE2_COORD1 = -3
 POSITIONS_2D = [
-    (POSITION_SAMPLE1, MOTOR_POSITION_SAMPLE1_COORD1, MOTOR_POSITION_SAMPLE1_COORD2),
-    (POSITION_SAMPLE2, MOTOR_POSITION_SAMPLE2_COORD1, MOTOR_POSITION_SAMPLE2_COORD2),
+    (POSITION_SAMPLE1, MOTOR_POSITION_SAMPLE1_COORD0, MOTOR_POSITION_SAMPLE1_COORD1),
+    (POSITION_SAMPLE2, MOTOR_POSITION_SAMPLE2_COORD0, MOTOR_POSITION_SAMPLE2_COORD1),
     ("Sample3", 0, -4),
     ("Sample4", 1, -4)
 ]
 
 
+def assert_alarm_state_of_posn(channel_access, coordinate, expected_state):
+    channel_access.assert_that_pv_alarm_is("FORWARD_ALARM", expected_state)
+    channel_access.assert_that_pv_alarm_is("POSN", expected_state)
+    channel_access.assert_that_pv_alarm_is("POSITIONED", expected_state)
+    channel_access.assert_that_pv_alarm_is("POSN_NO_ERR", ChannelAccess.Alarms.NONE)
+
+
 class MotionSetpointsTests(unittest.TestCase):
```

Comment:
Can we add a test that changes the definition file contents and then reloads the file by writing to RESET and check that the POSITIONS PV updates. Also, a test to see that if we change the contents to something invalid then the POSITIONS PV is empty or in error.